src/components/dapp-staking/DiscoverDappsTab.vue

```diff
@@ -92,13 +99,15 @@ export default defineComponent({
     const { accountData } = useBalance(currentAccount);
     const { stakeInfos } = useStakerInfo();
     const { dapps, stakingList } = useStakingList();
+    const { nativeTokenSymbol } = useNetworkInfo();
 
     const maxNumberOfStakersPerContract = computed(
       () => store.getters['dapps/getMaxNumberOfStakersPerContract']
     );
-    const minimumStakingAmount = computed(() => {
+
+    const minimumStakingAmount = computed<string>(() => {
       const amount = store.getters['dapps/getMinimumStakingAmount'];
-      return formatUnitAmount(amount);
+      return Number(ethers.utils.formatEther(amount)).toString() + ' ' + nativeTokenSymbol.value;
```

Comment:
Can we move these kinds of operations to astar.js? @niklabh

Reply:
ok